src/components/Header/Header.jsx

```diff
@@ -121,6 +125,7 @@ const Header = ({
       {hasSideNav && <HeaderMenuButton aria-label="Open menu" onClick={onClickSideNavExpand} />}
       <HeaderName href={url} prefix={prefix}>
         {appName}
+        {subtitle ? <div className={`${iotPrefix}--header__subtitle`}>{subtitle}</div> : null}
```

Comment:
Should we use the `HeaderMenuItem` from Carbon here instead? Might be a good idea so we are able to get any styling updates that might come upstream from Carbon. 

[Carbon story here for reference](https://react.carbondesignsystem.com/?path=/story/ui-shell--header-base-w-navigation)

Reply:
no only because this subtitle should be considered 'part of the headername and should be clickable as one.  It shouldn't take us to a separate landing page